[2023-08-01 22:56:58,615] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: s3_to_BigQuery.stations_data_to_BQ manual__2023-08-01T22:56:56.733909+00:00 [queued]>
[2023-08-01 22:56:58,626] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: s3_to_BigQuery.stations_data_to_BQ manual__2023-08-01T22:56:56.733909+00:00 [queued]>
[2023-08-01 22:56:58,636] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2023-08-01 22:56:58,636] {taskinstance.py:1357} INFO - Starting attempt 1 of 2
[2023-08-01 22:56:58,636] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2023-08-01 22:56:58,669] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): stations_data_to_BQ> on 2023-08-01 22:56:56.733909+00:00
[2023-08-01 22:56:58,675] {standard_task_runner.py:52} INFO - Started process 87 to run task
[2023-08-01 22:56:58,678] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 's3_to_BigQuery', 'stations_data_to_BQ', 'manual__2023-08-01T22:56:56.733909+00:00', '--job-id', '209', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmp3stq2ttn', '--error-file', '/tmp/tmpng1gbl6y']
[2023-08-01 22:56:58,679] {standard_task_runner.py:80} INFO - Job 209: Subtask stations_data_to_BQ
[2023-08-01 22:56:58,763] {task_command.py:370} INFO - Running <TaskInstance: s3_to_BigQuery.stations_data_to_BQ manual__2023-08-01T22:56:56.733909+00:00 [running]> on host 11cc9cd02b17
[2023-08-01 22:56:58,909] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=s3_to_BigQuery
AIRFLOW_CTX_TASK_ID=stations_data_to_BQ
AIRFLOW_CTX_EXECUTION_DATE=2023-08-01T22:56:56.733909+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-08-01T22:56:56.733909+00:00
[2023-08-01 22:56:58,910] {logging_mixin.py:115} INFO - 
 heeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeyy
[2023-08-01 22:56:59,406] {logging_mixin.py:115} INFO - 
✅ DataFrame created ! ✅
[2023-08-01 22:56:59,406] {logging_mixin.py:115} INFO - Error: [Errno 2] No such file or directory: '/opt/***/pluginslime-project-key.json'
[2023-08-01 22:56:59,409] {python.py:173} INFO - Done. Returned value was: None
[2023-08-01 22:56:59,455] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=s3_to_BigQuery, task_id=stations_data_to_BQ, execution_date=20230801T225656, start_date=20230801T225658, end_date=20230801T225659
[2023-08-01 22:56:59,532] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-08-01 22:56:59,579] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
